src/v0/destinations/google_adwords_offline_conversions/transform.js

```diff
@@ -134,9 +137,98 @@ const process = async (event) => {
   return response;
 };
 
+const getEventChunks = (event, storeSalesEvents, clickCallEvents) => {
+  const { message, metadata, destination } = event;
+  message.forEach((message) => {
+    if (message.body.JSON?.isStoreConversion) {
+      storeSalesEvents.push({ message, metadata, destination });
+    } else {
+      clickCallEvents.push(getSuccessRespEvents(message, [metadata], destination));
+    }
+  });
+  return { storeSalesEvents, clickCallEvents };
+};
+
```

Comment:
@ItsSudip  unit tests are missing for `getEventChunks` & `batchEvents` functions cc: @sanpj2292 
should take this as backlog item

Reply:
They are being covered as per the unit-tests of batching. But individual unit-tests can be taken as a backlog item.

Reply:
Yes & going forward refer the ones being written at `test/__tests__/data/` as component tests - They are not unit tests.

Reply:
sure